wiring/src/spark_wiring_ble.cpp

```diff
@@ -1001,10 +1002,24 @@ class BleServiceImpl {
         return endHandle_;
     }
 
+    bool& characteristicsDiscovered() {
```

Comment:
https://github.com/particle-iot/device-os/pull/1847#discussion_r310971495 :)

Nit: This is a really weird pattern when considering trivial types. I'd rather have this as a public member then or use setters/getters.